signac/contrib/project.py

```diff
@@ -707,32 +997,54 @@ def groupbydoc(self, key=None, default=None):
         If `key` is None, jobs are grouped by identity (by id), placing one job
         into each group.
 
-        :param key:
+        Parameters
+        ----------
+        key : str, iterable, or function
             The statepoint grouping parameter(s) passed as a string, iterable of strings,
             or a function that will be passed one argument, :attr:`Job.document`.
-        :type key:
-            str, iterable, or function
-        :param default:
+            (Default value = None)
+        default :
             A default value to be used when a given state point key is not present (must
             be sortable).
+
         """
         return self.find_jobs().groupbydoc(key, default=default)
 
     def to_dataframe(self, *args, **kwargs):
         """Export the project metadata to a pandas dataframe.
 
         The arguments to this function are forwarded to :py:meth:`.JobsCursor.to_dataframe`.
+
+        Parameters
+        ----------
+        *args :
+
+        **kwargs :
+
+
+        Returns
+        -------
+            class 'pandas.core.frame.DataFrame'
+
         """
         return self.find_jobs().to_dataframe(*args, **kwargs)
 
     def read_statepoints(self, fn=None):
         """Read all statepoints from a file.
 
-        :param fn: The filename of the file containing the statepoints,
+        See also :meth:`dump_statepoints` and :meth:`write_statepoints`.
```

Comment:
Use the "See Also" section mentioned here: https://numpydoc.readthedocs.io/en/latest/format.html#sections